docs/features/nullable-reference-types.md

```diff
@@ -136,18 +136,15 @@ var t = maybeNull; // t is !
 ```
 
 ### Suppression operator (`!`)
-The postfix `!` operator sets the top-level nullability to non-nullable.
+The postfix `!` operator sets the top-level nullability to non-nullable. Conversions on a suppressed expression produce no nullability warnings.
```

Comment:
Should we clarify if this "no nullability warnings" apply to non-top-level nullability or not here (second sentence)?